src/Symfony/Component/Clock/MockClock.php

```diff
@@ -80,19 +93,47 @@ public function modify(string $modifier): void
      */
     public function withTimeZone(\DateTimeZone|string $timezone): static
     {
+        $this->now ??= $this->tryFromTimeProvider();
```

Comment:
Do we actually need to initialize `now` here if it is not initialized yet ?

Reply:
We do not indeed, I removed this part